### PR TITLE
Update

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

File: src/window.rs [hunk 1]
```diff
@@ -113,6 +113,12 @@ impl Window {
                 attributes.get_mask() as c_int))
         }
     }
+
+    pub fn create_similar_surface(&self, content: cairo::Content, width: i32, height: i32) -> Option<Surface> {
```
Comment (EPashkin) [inline src/window.rs]: &cairo::Content ?

**Comment by EPashkin** [comment]
Maybe add `pango::Layout` too

File: src/window.rs (hunk 1 above)
Comment (EPashkin) [inline src/window.rs]: Oh, it enum.

**Comment by EPashkin** [comment]
`pango::Layout` generate only bad defined `gdk_pango_layout_get_clip_region`, so also not need

**Comment by EPashkin** [comment]
From errors:
`gdk_window_set_opaque_region` need `const` override.
`gdk_cairo_surface_create_from_pixbuf` has no `impl PixbufExt for Pixbuf`

**Comment by GuillaumeGomez** [comment]
Both fixed. So I didn't add `pango::Layout` as asked.

**Comment by EPashkin** [comment]
Lost `#[cfg(feature = "v3_10")]` for `impl SurfaceExt for Surface`

**Comment by GuillaumeGomez** [comment]
The function doesn't require it: https://developer.gnome.org/gdk3/stable/gdk3-Cairo-Interaction.html#gdk-cairo-region-create-from-surface

**Comment by EPashkin** [comment]
Then need `#[cfg(feature = "v3_10")]` need removed from `use cairo::Surface;` ;)

**Comment by GuillaumeGomez** [comment]
Oups, done!

**Comment by EPashkin** [comment]
LGFM 
but you don't against https://github.com/gtk-rs/glib/pull/196, IMHO better merge https://github.com/gtk-rs/glib/pull/196 first and update gir+regen in this PR.

**Comment by GuillaumeGomez** [comment]
@EPashkin: @sdroege told me he would update so I'll merge this one then wait for him to update.

**Comment by EPashkin** [comment]
k

**Pull request merged**